example.env

```diff
@@ -4,8 +4,16 @@ SERVER_PORT=4000
 # Slack
```

Comment:
To keep the naming consistent, let's use this. I don't think we'll be using those other ENVs. 

```
SLACK_BOT_OAUTH_TOKEN
SLACK_BOT_USER_ID
SLACK_BOT_CLIENT_ID
SLACK_BOT_CLIENT_SECRET
SLACK_BOT_CLIENT_SIGNING_SECRET
SLACK_BOT_REDIRECT_URL
SLACK_TEAM_ID
SLACK_BOT_VERIFICATION_TOKEN
```